resources/assets/js/videos/components/videoScreen/drawInteractions.vue

```diff
@@ -163,20 +164,22 @@ export default {
             this.$emit('pending-annotation', null);
         },
         extendPendingAnnotation(e) {
+            if (isInvalidShape(e.feature)) {
+                // Disallow shapes with too few points.
+                this.$emit('is-invalid-shape', e.feature.getGeometry().getType());
+                // Wait for this feature to be added to the source, then clear.
+                this.pendingAnnotationSource.once('addfeature', () => {
+                    this.resetPendingAnnotation();
+                });
+                // Reset mode to enable drawing again by closing button 
+                this.resetInteractionMode();
```

Comment:
This wasn't done before. Wouldn't it be easier if the button stayed active so the user can immediately try again?